### ISSUE TITLE
V7 Bug: Sys won't allow user to void $0 bills (NN) (Waiting on CTM/Victor to work)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#10221]
### This needs to go to CTM
- Example RSN 12970723
- Business Impact: 2
- User Izzy

<img width="619" alt="Screen Shot 2022-09-09 at 2 23 57 PM" src="https://user-images.githubusercontent.com/49930868/189428770-35adc864-2d2e-4652-8715-8a2b4ce1dc34.png">

<img width="1137" alt="Screen Shot 2022-10-11 at 3 57 14 PM" src="https://user-images.githubusercontent.com/49930868/195196990-0d9bdfa5-c691-4879-82d9-d01d63754ca9.png">

### Also cannot extend permits without voiding the payment

**Comment by TracyLinder** [comment]
We need to figure out if this is still a problem.

**Comment by Lpriya-LIT** [comment]
Update - Need to do a datafix to update outstanding balance for bill number #7656498 from 0.00 to 301.60. Then the bill can be void/cancel. 
@Lpriya-LIT will create a script and assign the ticket to dba.

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Created request - SCTASK0258668 to run the update script in Prod

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Script has been run in Prod. Please request User - Izzy to try cancelling this bill (7656498) in Prod folderrsn - 12970723.

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT Izzy tried again and got the same error:
![MicrosoftTeams-image (19).png](https://images.zenhubusercontent.com/5e8dfbfabf171441c3a5684e/d8a0347c-e2a7-41db-ab29-316f3fddcf98)

**Comment by TracyLinder** [comment]
Per Priya, AMANDA won't allow a bill of $0 to be voided. This needs to go to Jenice for the vendor to resolve. FYI @Nadin-Nader

**Comment by TracyLinder** [comment]
### Example

![Screenshot 2023-01-13 at 2.25.13 PM.png](https://images.zenhubusercontent.com/5cca1e522b8dfc0ee14dedf5/6ac785b6-193a-4e13-9a18-a10596a7d3a4)

**Comment by TracyLinder** [comment]
Per Izzy, Victor C. offered to help, just send him the ticket number.

**Comment by TracyLinder** [comment]
@Nadin-Nader please get with Victor. Thank you.

**Comment by Nadin-Nader** [comment]
This has been sent to CTM to be looked at under **SCTASK0281381**

**Comment by Nadin-Nader** [comment]
Scheduled a meeting for Wednesday 01/25/23 at 11am with Victor, Isabel and Priya

**Comment by TracyLinder** [comment]
I sent the email to CTMAmandaDL asking for an update. Thank you @Lpriya-LIT

**Comment by Lpriya-LIT** [comment]
@TracyLinder 
We just had a call with Victor regarding this issue. He mentioned that George has a fix for it , I will get in touch with George and provide you an update.
@Nadin-Nader

**Comment by TracyLinder** [comment]
@Nadin-Nader @Lpriya-LIT see my test results in PROD. System isn't updating the paid flag when the payment was voided, so i am unable to void the bill. I tried this same test with a $35 bill; when I voided the payment the payment flag updated. So this is a problem with the zero dollar balance. 

<img width="1240" alt="Screenshot 2023-01-25 at 3 27 03 PM" src="https://user-images.githubusercontent.com/49930868/214695221-b2ca884a-99e1-4e3c-a2de-1349b931a065.png">
